src/components/PostInfo/PostInfo.jsx

```diff
@@ -2,0 +1,21 @@
+import './PostInfo.scss';
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
+      </p>
+    </div>
+
+    <p className="PostInfo__body">{post.body}</p>
+
+    <CommentList comments={post.comments} />
```

Comment:
Ensure that the `CommentList` component is designed to accept a `comments` prop and render it correctly. The `post.comments` should be an array of comment objects.